### PR TITLE
Fix Deployment of Linked Contract

### PR DESCRIPTION
**Pull request by nlordell**
Fixes an issue with the generated deployment code for linked contracts

### Test Plan

New example that tests linked contracts

### CODE REVIEW

File: examples/linked.rs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    eloop.into_remote();
+    let web3 = Web3::new(http);
+
+    let library = SimpleLibrary::deploy(&web3)
+        .gas(4_712_388.into())
+        .confirmations(0)
+        .deploy()
+        .await
+        .expect("library deployment failure");
+    let instance = LinkedContract::deploy(&web3, library.address())
```
Comment (fleupold) [inline examples/linked.rs]: How do argument names for the constructor work. Do they come after or before the linked libraries?

**Review by fleupold — APPROVED** [review]
Should we run these examples as part of travis?

**Comment by nlordell** [comment]
@fleupold Adding examples to travis is coming soon to a PR near you :)

File: examples/linked.rs (hunk 1 above)
Comment (nlordell) [reply examples/linked.rs]: Good point! Maybe I should add this to the example.

**Pull request merged**